frontend/src/components/DateRangeFields.jsx

```diff
@@ -189,22 +194,26 @@ export default function DateRangeFields({
       {hasMultipleDates && (
         <button
           type="button"
-          className="btn btn-text text-link"
+          className="btn btn-text text-link p-0"
           onClick={() => addDateRange(dateType)}
         >
           <FontAwesomeIcon icon={faPlus} />
           <span className="ms-1">Add more {dateType.name} dates</span>
         </button>
       )}
 
-      <Form.Check
-        type="checkbox"
-        id={`date-range-annual-${dateType.id}`}
-        name={`date-range-annual-${dateType.id}`}
-        label="Dates are the same every year"
-        checked={isDateRangeAnnual}
-        onChange={handleDateRangeAnnualChange}
-      />
+      {/* Display checkbox except for Tier 1 and Tier 2 */}
+      {!(dateType.name === "Tier 1" || dateType.name === "Tier 2") && (
+        <Form.Check
+          type="checkbox"
+          id={`date-range-annual-${dateType.id}`}
+          name={`date-range-annual-${dateType.id}`}
+          label="Dates are the same every year"
+          checked={isDateRangeAnnual}
+          onChange={handleDateRangeAnnualChange}
+          className="mt-2 mb-0"
+        />
+      )}
     </>
```

Comment:
From the script, DateRangeAnnual was not supposed to be created if the date type is Tier 1/Tier 2. However, it's possible to add those from AdminJS, so I added another condition in the frontend.